[stable/mongodb-replicaset] helm lint --strict throws missing key error

**Version of Helm and Kubernetes**:
Helm v3.0.2
mongodb-replicaset v3.9.4 (also tried on latest master)

**Which chart**:
mongodb-replicaset

**What happened**:
Running `helm lint --strict` inside mongodb-replicaset chart throws the following error.

```
[ERROR] templates/: template: mongodb-replicaset/templates/_helpers.tpl:6:31: executing "mongodb-replicaset.name" at <.Values.nameOverride>: map has no entry for key "nameOverride"

Error: 1 chart(s) linted, 1 chart(s) failed
```

So it breaks charts which are dependent on mongodb-replicaset as well.
Looks like we need to add `nameOverride:""` in the values.yaml, similar to what is done here: 
https://github.com/helm/helm/pull/3434

--- a/stable/mongodb-replicaset/Chart.yaml
+++ b/stable/mongodb-replicaset/Chart.yaml
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: mongodb-replicaset
 home: https://github.com/mongodb/mongo
-version: 3.11.2
+version: 3.11.3
 appVersion: 3.6
 description: NoSQL document-oriented database that stores JSON-like documents with
   dynamic schemas, simplifying the integration of data in content-driven applications.

--- a/stable/mongodb-replicaset/values.yaml
+++ b/stable/mongodb-replicaset/values.yaml
@@ -1,3 +1,7 @@
+# Override the name of the chart, which in turn changes the name of the containers, services etc.
+nameOverride: ""
+fullnameOverride: ""
+
 replicas: 3
 port: 27017
 
@@ -12,14 +16,14 @@ podDisruptionBudget: {}
 
 auth:
   enabled: false
+  existingKeySecret: ""
+  existingAdminSecret: ""
+  existingMetricsSecret: ""
   # adminUser: username
   # adminPassword: password
   # metricsUser: metrics
   # metricsPassword: password
   # key: keycontent
-  # existingKeySecret:
-  # existingAdminSecret:
-  # existingMetricsSecret:
 
 ## Optionally specify an array of imagePullSecrets.
 ## Secrets must be manually created in the namespace.
@@ -99,7 +103,7 @@ tolerations: []
 
 extraLabels: {}
 
-# priorityClassName: ""
+priorityClassName: ""
 
 persistentVolume:
   enabled: true
@@ -110,7 +114,7 @@ persistentVolume:
   ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
   ##   GKE, AWS & OpenStack)
   ##
-  # storageClass: "-"
+  storageClass: ""
   accessModes:
     - ReadWriteOnce
   size: 10Gi
@@ -144,6 +148,7 @@ configmap: {}
 # initMongodStandalone: |+
 #   db = db.getSiblingDB("mydb")
 #   db.my_users.createIndex({email: 1})
+initMongodStandalone: ""
 
 # Readiness probe
 readinessProbe:

